spec/requests/certificates/i_belong_dashboard_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe Certificates::IBelongCertificateController do
+  let(:user) { create(:user) }
+  let(:certificate) { create(:i_belong_certificate) }
+  let(:enrolment) { create(:user_programme_enrolment, user: user, programme: certificate) }
+
+  describe '#show' do
+    before do
+      stub_attendance_sets
+      stub_delegate
+    end
+
+    describe 'while unenrolled' do
+      before do
+        certificate
+        allow_any_instance_of(AuthenticationHelper).to receive(:current_user).and_return(user)
+      end
+
+      it 'redirects to I Belong public page' do
+        skip 'public page branch not merged yet, so i_belong_path not defined'
+
```

Comment:
```suggestion
        pending 'public page branch not merged yet, so i_belong_path not defined'

```
I think I meant 'pending' here, because when the i-belong public page branch is merged, this spec will automatically fail and the pending line can be removed to make it pass (Hopefully someone will also notice the TODOs at the same time but 🤷 )